internal/controller/controllers/agentserviceconfig_controller_test.go

```diff
@@ -1156,6 +1264,21 @@ var _ = Describe("ensureAssistedServiceDeployment", func() {
 			})
 		})
 	})
+
+	Describe("ipxe settings", func() {
```

Comment:
I don't think you need an extra `Describe` here.
Since it's one test you can just include the context in the name `should expose two ports for ipxe`, would work.

Also, semantically this would be a `Context` rather than a `Describe` but since you're not actually adding any context here I would also suggest just changing the test name in that case.